jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -26,6 +27,17 @@
 from jax import numpy as jnp
 
 class JaxToTfTestCase(jtu.JaxTestCase):
+  @classmethod
+  def setUpClass(cls):
+    # Initialize categorizer
+    categorizer = jax2tf.jax2tf.Categorizer()
+    # Register limitation summary atexit printing handler
+    atexit.register(categorizer.pprint_limitations, None)
```

Comment:
We must have a flag or environment variable to guard the `atexit.register`. I generally prefer an environment variable for something that needs to be checked only once. There are going to be executions when we do not want the atexit handler to run, e.g., inside automated tests in Google (any kind of file writing will fail unless it is directed to some special directories), or when we run selected tests manually for debugging.

Reply:
Yes, this was always the final intent; I'll add it now.